src/platform_impl/macos/view.rs

```diff
@@ -317,6 +317,8 @@ extern "C" fn view_did_move_to_window(this: &Object, _sel: Sel) {
             assumeInside:NO
         ];
         state.tracking_rect = Some(tracking_rect);
+
+        // TBD: Do we need to emit a resize event here like we do in frame_did_change?
```

Comment:
I'm not sure. But you can check it by looking through logs in various cases and see if it's called during 'tab' related stuff and whether the size is correct. If it always called once, maybe we should resize here and not in frameDidChange. Also, I'd assume that every resize is followed by frameDidChange or prefix by it. So we could remove resize from windowDidResize?

Reply:
Ok, I did some testing and yes, the viewDidMoveToWindow is called when creating and deleting (tabbed) window, but it has the wrong size for new tabbed windows.  So it looks like we don't want or need this one.  I'll go ahead and remove the comment.

You are correct that every windowDidResize is prefixed or followed by a frameDidChange.  For normal resizing with the mouse, frameDidChange comes first.  For a new window, if windowDidResize gets called at all, the frameDidChange still comes first, however in the case of a tabbed window, frameDidChange also gets called after and that is the one that has the correct value we need.

I get the feeling that when creating a new tabbed window, macOS behaves as if it first exists without the tab bar (so has a taller height), and then an instant later decides that it is being placed in a tab so then it evokes another frameDidChange with the correct size (that excludes the tab height).

Anyway, regarding windowDidResize I think you're right we can actually remove emitting a resize from there now.  Shall I do that?

Reply:
>Anyway, regarding windowDidResize I think you're right we can actually remove emitting a resize from there now. Shall I do that?

Yeah, go ahead.